.circleci/config.yml

```diff
@@ -1,16 +1,41 @@
 version: 2
 jobs:
-  build:
-    docker:
-      - image: rocker/tidyverse:3.6.0
-
+  run_tests:
+    machine:
+      docker_layer_caching: true
+    working_directory: ~/OpenPBTA-analysis
     steps:
       - checkout
 
       - run:
-          name: Greeting
-          command: echo Hello, world.
+          name: Data Download
+          command: URL=https://open-pbta.s3.amazonaws.com/data RELEASE=testing ./download-data.sh
 
       - run:
-          name: Print the Current Time
-          command: date
+          name: Analysis 1
```

Comment:
What do you think about naming this something more specific? Just thinking out loud here that we might want folks who add their own analyses to have a non-numeric name.
> List Data Directory
or something